installer/win/Symphony-x64.aip

```diff
@@ -521,6 +504,13 @@
     <ROW Dialog_="SettingsDlg" Control="Title" Type="Text" X="15" Y="6" Width="200" Height="15" Attributes="196611" Text="Basic Settings" TextStyle="[DlgTitleFont]" Order="1800" TextLocId="Control.Text.FolderDlg#Title"/>
     <ROW Dialog_="SettingsDlg" Control="Description" Type="Text" X="17" Y="21" Width="272" Height="14" Attributes="196611" Text="Select features for your users." Order="1900" TextLocId="Control.Text.FolderDlg#Description"/>
     <ROW Dialog_="SettingsDlg" Control="PushButton_1" Type="PushButton" X="267" Y="206" Width="92" Height="17" Attributes="3" Text="Advance Settings" Order="2000"/>
+    <ROW Dialog_="UrlValidationDlg" Control="Next" Type="PushButton" X="236" Y="243" Width="56" Height="17" Attributes="1" Text="[ButtonText_Next]" Order="100" TextLocId="-" Options="1"/>
+    <ROW Dialog_="UrlValidationDlg" Control="Cancel" Type="PushButton" X="304" Y="243" Width="56" Height="17" Attributes="2" Text="[ButtonText_Cancel]" Order="200" TextLocId="-" Options="1"/>
+    <ROW Dialog_="UrlValidationDlg" Control="Bitmap" Type="Bitmap" X="0" Y="0" Width="370" Height="234" Attributes="1048577" Text="[DialogBitmap]" Order="300"/>
+    <ROW Dialog_="UrlValidationDlg" Control="Back" Type="PushButton" X="180" Y="243" Width="56" Height="17" Attributes="3" Text="[ButtonText_Back]" Order="400" TextLocId="-" Options="1"/>
+    <ROW Dialog_="UrlValidationDlg" Control="Title" Type="Text" X="135" Y="20" Width="220" Height="60" Attributes="196611" Text="Invalid POD URL" TextStyle="VerdanaBold13" Order="500"/>
+    <ROW Dialog_="UrlValidationDlg" Control="Description" Type="Text" X="135" Y="86" Width="220" Height="60" Attributes="196611" Text="You seem to have entered an invalid pod url. Please go back to the previous screen and rectify it." Order="600"/>

```

Comment:
As far as I am aware of, we are doing a "generic" URL validation and not necessarily validating if the URL entered is a valid "POD" URL. If that's the case, the copy on this has to change to something along the lines of `You seem to have entered an invalid URL...` instead of `You seem to have entered an invalid Pod Url...`.

Only if the user is unable to enter anything apart from the POD URLs supported by Symphony will this copy hold true. If the user is able to input, for ex., `http://www.apple.com`, then this copy won't hold true.

Please talk to Nancy and Noah for the final copy on this.

CC: @VishwasShashidhar

Reply:
@VikasShashidhar Sure will add a comment in the ticket 😃

Reply:
this was approved back in the day by Eric from design. Seeing that this is an extremely important aspect of the installation process, I'd say that we should go with this PR and create another ticket for changing the design.

Reply:
@VikasShashidhar Updated the text content as provided by Nancy 😃